### PR TITLE
Trying out git commit

### PR DESCRIPTION
**Pull request by sherryyang2016**

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -66,5 +66,25 @@
       </p>
     </div>
   </div>
+  
+<div id="container">
+	<div
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Make sure to check out the html style guide here:   https://google.github.io/styleguide/htmlcssguide.html
There's a bit of indentation mismatches here.

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -66,5 +66,25 @@
...
+       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+       
+       <h1>Team 21 Info</h1>
+       
+       <ul>
+         <li><strong>(This is just a git operation test XD )</strong>
+         </li>
+         <li><strong>Sherry Yang</strong> I am a sophomore studying at University of Washington,
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Can you deploy your change locally and attach a screenshot of the new page for UI changes?  Thanks :)

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -66,5 +66,25 @@
...
+         	Seattle (yes, I crave for sunlight). So far I have been gaining backend skills and I
+         	do hope for getting involved in the "frontend community." One interesting fact about 
+         	myself:I have been (too) into the kpop band BTS since college, it is sometimes unhealthy
+         	to watch countless youtube vidoes about them ^o^.  
+         </li>
+       </ul>
+     </div>
+   </div>
+
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Unnecessary line.

**Review by turtlefarmer — CHANGES_REQUESTED** [review]

File: src/main/webapp/about.jsp (hunk 2 above)
Comment (sherryyang2016) [reply src/main/webapp/about.jsp]: image

File: src/main/webapp/about.jsp (hunk 1 above)
Comment (sherryyang2016) [reply src/main/webapp/about.jsp]: I will take a look, thank you! It is totally new for me to work with html so I might make a lot of mistakes.

**Pull request closed without merging**